config/notifiers.go

```diff
@@ -394,3 +401,45 @@ func (c *PushoverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	}
 	return checkOverflow(c.XXX, "pushover config")
 }
+
+// StatusPageConfig configures notifications via StatusPage.
+type StatusPageConfig struct {
+	NotifierConfig `yaml:",inline"`
+
+	APIURL      string `yaml:"api_url"`
+	APIKey      Secret `yaml:"api_key"`
+	PageId      string `yaml:"page_id"`
+	ComponentId string `yaml:"component_id"`
+	Status      string `yaml:"status"`
```

Comment:
Those parameters are only available when you [create/update an incident](https://doers.statuspage.io/api/v1/incidents/).